CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 ## Unreleased:
 
+## Version 1.0.0 - 2021-02-26
+
+### General
```

Comment:
Would be good to maybe have a link to the stable packages?

Reply:
good idea, yes... can't really add that until we do a publish and we can point at a published bom/pom, though, i think.

Reply:
Ok, probably we can do it in the release notes. Or we can have an entry to the stable bom in the readme :) Can be done later.

Reply:
I added it here.